packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -995,6 +995,11 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         const pendingRuntimeState = context.pendingLocalState as IPendingRuntimeState | undefined;
         const baseSnapshot: ISnapshotTree | undefined = pendingRuntimeState?.baseSnapshot ?? context.baseSnapshot;
 
+        //* Test to prove this doesn't get persisted as undefined?
```

Comment:
This is a todo, might or might not do it, it would be an e2e test. When this is undefined due to storage not existing yet, it shouldn't be persisted because only Summarizer persists these values.